Warnings because of ini-parser

## Describe the bug
We currently get some warnings because we're using [ini-parser](https://www.nuget.org/packages/ini-parser/) which is not packaged for .NET Standard.

We should instead use [ini-parser-netstandard](https://www.nuget.org/packages/ini-parser-netstandard/) to avoid the warnings.

## To Reproduce
Steps to reproduce the behavior:
1. Restore nuget-packages for AltinnCore 
2. See warning:
> warning NU1701: Package 'ini-parser 2.5.2' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.1'. This package may not be fully compatible with your project."

## Expected behavior
No warnings.

--- a/src/AltinnCore/Common/AltinnCore.Common.csproj
+++ b/src/AltinnCore/Common/AltinnCore.Common.csproj
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="HtmlAgilityPack" Version="1.8.12" />
-    <PackageReference Include="ini-parser" Version="2.5.2" />
+    <PackageReference Include="ini-parser-netstandard" Version="2.5.2" />
     <PackageReference Include="Manatee.Json" Version="10.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.0" />
     <PackageReference Include="Microsoft.CodeAnalysis" Version="2.8.2" />

--- a/src/AltinnCore/Designer/AltinnCore.Designer.csproj
+++ b/src/AltinnCore/Designer/AltinnCore.Designer.csproj
@@ -22,7 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="ini-parser" Version="2.5.2" />
+    <PackageReference Include="ini-parser-netstandard" Version="2.5.2" />
     <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.0" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.0" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />

